etc/conf.templates/standalone/00-warp.conf.template

```diff
@@ -252,6 +257,13 @@ warp.token.read.attributes.default = { '.cap:limits' '' }
 //
 #warp.attributes.maxsize = 8192
 
+//
+// When using a standalone version with no replication to a FoundationDB based instance
+// the limit of metadata size (class + attribtues + labels) can be relaxed by setting this
+// configuration to 'true' to simply emit a warning.
+//
+#warp.relax.metadata.maxsize = true
+
```

Comment:
Typo on attributes, and may be rephrased:

_By default, metadata size is limited to the maximum allowed by fdb.
When using a leveldb standalone version, the limit of metadata size (class + attributes + labels) can be relaxed by setting this configuration to 'true' to simply emit a warning.
This parameter has no effect on standalone+ (fdb backend).
Relaxing the limit means that data cannot be copied to an fdb backend in the future._